test/manual/amp-selector.amp.html

```diff
@@ -0,0 +1,120 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>AMP #0</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <script async custom-element="amp-form" src="../../dist/v0/amp-form-0.1.max.js"></script>
+  <script async custom-element="amp-selector" src="../../dist/v0/amp-selector-0.1.max.js"></script>
+  <script async custom-element="amp-carousel" src="../../dist/v0/amp-carousel-0.1.max.js"></script>
+  <style amp-custom>
+  /* AMP.CSS */
+  html, body {
```

Comment:
this is just an example file. not the css file - the component does not need any css